lib/main.rb

```diff
@@ -1,27 +1,30 @@
 require_relative "./product"
-require_relative "./warehouse"
 require_relative "./basket"
 
-p1 = Product.new(name: "Football", price: 79)
-p2 = Product.new(name: "Basketball", price: 60)
-p3 = Product.new(name: "Squash racket type A", price: 150)
-p4 = Product.new(name: "Squash racket type B", price: 185.60)
-p5 = Product.new(name: "Sports bag", price: 145)
+begin
+  p1 = Product.new(name: "Football", price: 79.44)
+  p2 = Product.new(name: "Basketball", price: 60)
+  p3 = Product.new(name: "Squash racket type A", price: 150)
+  p4 = Product.new(name: "Squash racket type B", price: 185.60)
+  p5 = Product.new(name: "Sports bag", price: 145)
 
-basket = Basket.new
+  basket = Basket.new
 
-basket.add(p1)
-basket.add(p2)
-basket.add(p2)
-basket.add(p2)
-basket.add(p3)
+  basket.add(p1)
+  basket.add(p2)
+  basket.add(p2)
+  basket.add(p2)
+  basket.add(p3)
 
-p basket.show
+  p basket.show
```

Comment:
Okay.. it is confusing now, because `p` here gets the value from [`puts sum_with_vat` (`nil`)](https://github.com/shadrk/shop/pull/1/files#diff-3d09b648c0559f43626ac46a9083d7a0R41) )(https://github.com/shadrk/shop/pull/1/files#r70170165). `Basket#show` can be renamed to `Basket#details` or `Basket#all` and it should return data. Then you can present them here (with `each` and `puts` etc)